netbout-client/src/test/java/com/netbout/client/NbRule.java

```diff
@@ -45,6 +45,7 @@
     /**
      * Get user.
      * @return User
+     * @checkstyle NonStaticMethodCheck (10 lines)
```

Comment:
@danff if method does not refer to any instance field or method then it should be static